modules/programs/dircolors.nix

```diff
@@ -203,21 +211,21 @@ in {
       ".xspf" = mkDefault "00;36";
     };
 
-    home.file.".dir_colors".text = concatStringsSep "\n" ([ ]
+    home.file.${cfg.path}.text = concatStringsSep "\n" ([ ]
```

Comment:
It would be good to add a test to this to make sure that it's putting the file in the correct place.